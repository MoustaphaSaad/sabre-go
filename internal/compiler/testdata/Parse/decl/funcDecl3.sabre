func (s string, r rune) int