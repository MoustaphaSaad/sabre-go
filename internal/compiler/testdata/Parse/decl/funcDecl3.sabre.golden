>> 	func (s string, r rune) int
>> 	     ^^^^^^^^^^^^^^^^^^     
Error[internal/compiler/testdata/Parse/decl/funcDecl3.sabre:1:6]: method is expected to have only one receiver
